kernal/cbm/editor.s

```diff
@@ -505,7 +505,7 @@ wlogic
 	cmp pntr        ;if lnmx is less than pntr
 	bcs wlgrts      ;branch if lnmx>=pntr
 	cmp #maxchr-1   ;past max characters
-	beq wlog10      ;branch if so
+	bcs wlog10      ;branch if so
```

Comment:
So this was backwards from what it needed to be? Or is this a tweak made for debugging line truncation?

Reply:
Wait, nevermind, I'm dumb and misread it the first time.